data/vector.go

```diff
@@ -72,12 +72,14 @@ func (v Vector) Mod(modulo *big.Int) Vector {
 	return NewVector(newCoords)
 }
 
-// CheckBound checks whether all vector elements are strictly
-// smaller than the provided bound.
-// It returns error if at least one element is >= bound.
+// CheckBound checks whether the absolute value of all vector elements
+// are strictly smaller than the provided bound.
```

Comment:
value -> values :)

Reply:
Ok, I will go with values since CS people will read this, but if you think of an absolute value as an operator, then there is only one such operator :)